cli/compile/index.js

```diff
@@ -57,9 +57,9 @@ function handler(argv) {
         globs: argv.globs,
         reporter: argv.reporter
       }),
-      tasks = [fonts, styles, templates, scripts],
+      tasks = [scripts],
       builders = _.map(tasks, (task) => task.build),
-      watchers = _.map(tasks, (task) => task.watch).concat([media.watch]),
+      watchers = _.map(tasks, (task) => task.watch),
```

Comment:
in this file, we're just stripping out all build tasks except for `scripts`, to make compilation as fast as possible for developers who are only touching `.vue` files